src/adapter/src/session.rs

```diff
@@ -90,6 +91,7 @@ where
     // is also used to return an error when no role exists and
     // therefore there is no valid `RoleMetadata`.
     role_metadata: Option<RoleMetadata>,
+    peer_address: Option<IpAddr>,
```

Comment:
I'd suggest calling this `client_address` everywhere in our codebase, to align with what PostgreSQL calls the column in their public interface, and therefore what I think we should call the column in our public interface (`mz_sessions.client_address`).

```suggestion
    client_address: Option<IpAddr>,
```